src/domain/app/i18n/en.json

```
@@ -1887,8 +1887,8 @@
     },
     "personsAddedToWaitingListModal": {
       "buttonClose": "Close",
-      "text": "Please note that not all registrants' places are guaranteed. Entrants in the reserve position will only be confirmed if more places become available in the queue. You will receive a separate email from confirmed registration.",
-      "title": "Entrants have been added to the reservation queue"
+      "text": "You will receive a separate confirmation message about your registration to your e-mail, if a place becomes available for you.",
```

```suggestion
      "text": "You will receive a separate confirmation message about your registration to your e-mail if a place becomes available for you.",
```
The English grammar rules are so that [a comma isn't normally used](https://www.kotus.fi/nyt/uutistekstit/uutiskirjeet/kielikuulumisia-uutiskirjeiden_arkisto/4_2013/oikein_siella_vaarin_taalla) in this kind of a "päälause + sivulause" case

---

Changed as suggested. I did similar change to Linked Registration UI in https://github.com/City-of-Helsinki/linkedregistrations-ui/pull/93/files